stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -148,127 +149,125 @@ impl FromSql for HeaderTypeNames {
+                    burn_spent INTEGER NOT NULL,
+                    -- the consensus hash of the burnchain block that selected this block's miner's block-commit
+                    consensus_hash TEXT NOT NULL,
+                    -- the parent StacksBlockId
+                    parent_block_id TEXT NOT NULL,
+                    -- Merkle root of a Merkle tree constructed out of all the block's transactions
+                    tx_merkle_root TEXT NOT NULL,
+                    -- root hash of the Stacks chainstate MARF
+                    state_index_root TEXT NOT NULL,
+                    -- miner's signature over the block
+                    miner_signature TEXT NOT NULL,
+                    -- signers' signatures over the block
+                    signer_signature TEXT NOT NULL,
+                    -- bitvec capturing stacker participation in signature
+                    signer_bitvec TEXT NOT NULL,
+        -- The following fields are not part of either the StacksHeaderInfo struct
+        --   or its contained NakamotoBlockHeader struct, but are used for querying
+                    -- what kind of header this is (nakamoto or stacks 2.x)
+                    header_type TEXT NOT NULL,
+                    -- hash of the block
+                    block_hash TEXT NOT NULL,
+                    -- index_block_hash is the hash of the block hash and consensus hash of the burn block that selected it,
+                    -- and is guaranteed to be globally unique (across all Stacks forks and across all PoX forks).
+                    -- index_block_hash is the block hash fed into the MARF index.
+                    index_block_hash TEXT NOT NULL,
+                    -- the ExecutionCost of the block
+                    cost TEXT NOT NULL,
+                    -- the total cost up to and including this block in the current tenure
+                    total_tenure_cost TEXT NOT NULL,
+                    -- this field is true if this is the first block of a new tenure
+                    tenure_changed INTEGER NOT NULL,
+                    -- this field tracks the total tx fees so far in this tenure. it is a text-serialized u128
+                    tenure_tx_fees TEXT NOT NULL,
+                    -- nakamoto block's VRF proof, if this is a tenure-start block
+                    vrf_proof TEXT,
+
+            PRIMARY KEY(consensus_hash,block_hash)
+        );
+        CREATE INDEX nakamoto_block_headers_by_consensus_hash ON nakamoto_block_headers(consensus_hash);
+"#.into(),
+format!(
+    r#"ALTER TABLE payments
+        ADD COLUMN schedule_type TEXT NOT NULL DEFAULT "{}";
+    "#,
+    HeaderTypeNames::Epoch2.get_name_str()),
+r#"
+UPDATE db_config SET version = "4";
+"#.into(),
+]
+});
 
-    pub static ref NAKAMOTO_CHAINSTATE_SCHEMA_2: Vec<String> = vec![
-    NAKAMOTO_TENURES_SCHEMA_2.into(),
+pub static NAKAMOTO_CHAINSTATE_SCHEMA_2: &[&str] = &[
```

Comment:
Can be `const` I think

```suggestion
pub const NAKAMOTO_CHAINSTATE_SCHEMA_2: &[&str] = &[
```

Reply:
Yeah, missed it, good catch